src/main/java/bio/terra/common/SynapseColumn.java

```diff
@@ -43,6 +46,10 @@ public SynapseColumn requiresJSONCast(boolean requiresJSONCast) {
     return this;
   }
 
+  public boolean getIsFileType() {
+    return FILE_TYPES.contains(getType());
```

Comment:
Maybe this should be `isDrsType()` since it's both file and dir types, and what we're using it for is to determine if it needs to be converted to a DRS URL.

Reply:
I don't want it to be drs specifically since we may be allowing generating different types of file uris.  At this point, it's really just that it's a file